assets/queries/k8s/block_privileged_container/query.rego

```diff
@@ -0,0 +1,15 @@
+package Cx
+
+CxPolicy [ result ] {
+   spec := input.document[i].spec 
+   containers := spec.containers
+   containers[c].securityContext.privileged == true 
+   
+   result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("spec.containers[%d].securityContext.privileged", [c]),
+                "issueType":		   "IncorrectValue",
+                "keyExpectedValue": sprintf("spec.containers[%d].securityContext.privileged is false", [c]),
+                "keyActualValue": 	sprintf("spec.containers[%d].securityContext.privileged is true", [c])
+              }
+}
```

Comment:
Check this modifications, we have to make sure that the searchKey is pointing to the right place:

package Cx

CxPolicy [ result ] {
   metadata:= input.document[i].metadata
   spec := input.document[i].spec 
   containers := spec.containers
   containers[c].securityContext.privileged == true 

   result := {
                "documentId": 		input.document[i].id,
                "searchKey": 	    sprintf("metadata.name=%s.spec.containers.name=%s.securityContext.privileged", [metadata.name, containers[c].name]),
                "issueType":		   "IncorrectValue",
                "keyExpectedValue":  sprintf("spec.containers.name=%s.securityContext.privileged is false", [metadata.name, containers[c].name]),
                "keyActualValue": 	 sprintf("spec.containers.name=%s.securityContext.privileged is true", [metadata.name, containers[c].name])
              }
}